nlpsandboxclient/cli/community.py

```diff
@@ -22,9 +22,12 @@ def get_num_users():
 
 @cli.command()
 @click.option('--output', help='Output json filepath', type=click.Path())
-def get_clinical_notes(output):
+@click.option('--data_endpoint', help='Data endpoint')
```

Comment:
- Do not use "data_" as a synonym of "data_node_"

The term `endpoint` is usually used to refer to the full URL of a request, i.e. nothing expected after the _end_point. Here I think that you are using this information as the base of the endpoint.

The OpenAPI provides some hints regarding how we should name this term.

![image](https://user-images.githubusercontent.com/3056480/96536495-1211c880-1249-11eb-9202-d011fd7a4ba3.png)

[Source](https://swagger.io/docs/specification/2-0/api-host-and-base-path/)

Let's have a live chat to settle on a naming convention given that we will use them terms countless times in our projects. I would like also to identify how best handle this information when this package is used with the CLI and as a python dependency. I'll setup a brief call.

Reply:
Ya, I have always struggled with the terminology.  One thing we can technically do is break it down into three, what we are looking for is the `scheme`, `host` and `basePath`.

Reply:
Decided to use `data_node_host`